src/EventStore.Core/ClusterVNodeOptions.cs

```diff
@@ -788,8 +789,8 @@ public record ProjectionOptions {
 			[Description("The time in milliseconds allowed for the compilation phase of user projections")]
 			public int ProjectionCompilationTimeout { get; set; } = 500;
 
-			[Description("The time in milliseconds allowed for the executing a handler in a user projection")]
-			public int ProjectionExecutionTimeout { get; set; } = 250;
+			[Description("The time in milliseconds allowed for the executing a handler in a user projection. It can be overriden for a specific projection by setting ProjectionExecutionTimeout config for that projection")]
```

Comment:
```suggestion
			[Description("The maximum execution time in milliseconds for executing a handler in a user projection. It can be overridden for a specific projection by setting ProjectionExecutionTimeout config for that projection")]
```